docs/general/ad-filtering/create-own-filters.md

```diff
@@ -425,6 +425,19 @@ This modifier lets you narrow the rule coverage down to a specific application (
 
 For Mac, you can find out the bundle ID or the process name of the app by viewing the respective request details in the Filtering log.
 
+**Syntax**
+
+The modifier is a list of one or more expressions separated by the `|` symbol, each of which is matched against a application in a particular way depending on its type (see below).
+
+```text
+applications = ["~"] entry_0 ["|" ["~"] entry_1 ["|" ["~"]entry_2 ["|" ... ["|" ["~"]entry_N]]]]
+entry_i = ( regular_app / any_tld_app / regexp )
+```
+
+- **`regular_app`** — a regular application name (`example.app`). Corresponds the specified application and its subapplications. It is matched lexicographically.
+- **`any_tld_app`** — a application name ending with a wildcard character as a [public suffix](https://publicsuffix.org/learn/), e.g. for `example.*` it is `app` in `example.app`. Corresponds to the specified application and its subapplications with any public suffix. It is matched lexicographically.
```

Comment:
It is unclear how the concept of a "subapplication" is defined and how the concept of a public suffix relates to an application name. I feel like this PR tries to apply domain restriction logic to application restriction, which is incorrect: even if application names may in some cases look like domain names (e.g., bundle IDs on macOS), they are not.